espnet2/asr/decoder/linear_decoder.py

```diff
@@ -0,0 +1,98 @@
+"""A simple linear layer decoder.
```

Comment:
This seems to be very related to spk-style. I'm totally fine with the current setup, but any options to connect that?

Reply:
I have prepared the code for cls1 task (containing multi-class and multi-label classification) and will prepare another PR for it later.

Reply:
The plan is to move code from this recipe to the task introduced here -> https://github.com/espnet/espnet/pull/5998